### PR TITLE
[WD-16470] Page refresh microcloud

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Copy update
- Changed Youtube Link
- Added a new quote section
- Update logo assets

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/microcloud
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Make sure that the page matches the design: https://www.figma.com/design/hzQYJM9Tui4wpmyZC0k0WS/23.10-C.com---MicroCloud-Rebranding?node-id=0-1&node-type=canvas&t=gTy0rbPmisXRTtBj-0
- Make sure that the changes in this copy doc are applied: https://docs.google.com/document/d/17iBbFCINLYjPBOYWpgoq9SreyYNkR0Sb5jgx5iTo6MA/edit?tab=t.0

## Issue / Card

Fixes WD-16470

### CODE REVIEW

**Comment by akbarkz** [comment]
@goulinkh I didn't understand why did the Vanilla patterns appear in this PR? Did you rebase it against the latest main branch?

**Comment by goulinkh** [comment]
@akbarkz I've removed the new macros, I did a rebase half way through which didn't show conflicts for macros due to the naming difference `macros`, `_macros`.

The original macro `vf_quoted_wrapper` that I added had some changes to match the design, we may need to make adjustments to the current one `quoted_wrapper` if the designers review it as -1.

**Comment by mattea-turic** [comment]
Thanks @goulinkh !

Just a couple of things, even though this was technically only a copy update (but I think it's still worth doing these):

- `<br>` after "is" in "What is a Microcloud?" please
- `<br>` after "scalable" in "fast and scalable cloud infrastructure"
- Muted rule here pls:
Screenshot 2024-11-15 at 09 04 23

- `<br>` after "and" in the key feature, "Remote authentication, fine grained access control, projects and multi-user setup." of the "Key features" section
- For the three sections: "Edge computing at a large scale", "Scalable cloud infrastructure", and "Efficient dev environment", could the rules above the ctas be muted?
- "Edge computing..." and "Scalable cloud..." should also have regular bottom padding following the cta, like you've done for "Efficient dev environment"
- `<br>` after "laptop" in the paragraph "MicroCloud is lightweight enough that it can run on a developer laptop. You can use it for:" in the section "Efficient dev environment"
- I believe the tag where in the header where "Microcloud" sits in the nav should maybe be the product tag, but I'd like to verify this with @Sophie-32 or @akbarkz

**Comment by goulinkh** [comment]
Thanks for the review @mattea-turic I've applied the suggested changes (except the last point).

**Comment by mattea-turic** [comment]
Looks great, thank you @goulinkh 

I'll add a +1 for design – and we'll see what the outcome of the product tag will be once the others get back to me

**Comment by Sophie-32** [comment]
missing a copy edit in this section - 
Screen Shot 2024-11-15 at 11 05 44 AM

**Comment by goulinkh** [comment]
Thanks for reporting this @Sophie-32, I've fix the spelling to US.

**Comment by lyubomir-popov** [comment]
This copy is a lot - can we please switch to  h6 instead of h4? This will take it down to paragraph fonti-size, and reduce scrolling

**Comment by mattea-turic** [comment]
@goulinkh Update on the tag - we should be using the tag that comes with the product logo, so in this case it would be the one with the cloud. I'm not sure how replacing this works on the dev side - do you know where the asset would be for this or should I provide you with a file of a specific format?

**Comment by goulinkh** [comment]
I've made the changes could you review the changes based on your feedback please?

**Comment by mattea-turic** [comment]
@goulinkh Looks good, thanks (and appreciate how responsive you are with these changes)

**Comment by akbarkz** [comment]
@goulinkh there was one small error from djLint reported in CI, phal

File: templates/microcloud/index.html [hunk 1]
```diff
@@ -1,405 +1,514 @@
...
+        </p>
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Lightweight and constrained disposable testing environments.</li>
+          <li class="p-list__item is-ticked">Simulating or testing complex infrastructure processes.</li>
+          <li class="p-list__item is-ticked">Simulating how your workloads would run in production.</li>
+          <li class="p-list__item is-ticked">Safely experimenting with new technologies.</li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <p>
+          <a href="https://canonical-microcloud.readthedocs-hosted.com/en/stable/tutorial/">Deploy your own MicroCloud following this tutorial&nbsp;&rsaquo;</a>
```
Comment (akbarkz) [inline templates/microcloud/index.html:387]: For me this link did not open, I got 404. I tagged the stakeholder in the copy doc about this, not a blocker.

**Review by akbarkz — APPROVED** [review]

**Pull request merged**